Dockerfile

```diff
@@ -44,9 +44,16 @@ CMD ["-a", "build"]
 #
 # DOCKER_BUILDKIT=1 docker build --target go --tag ockam/tool/go:latest .
 # docker run --rm --volume "$(pwd):/project" ockam/tool/go:latest
-FROM golang:1.11.2-alpine3.8@sha256:692eff58ac23cafc7cb099793feb00406146d187cd3ba0226809317952a9cf37 as go
+FROM golang:1.11.2-alpine3.8@sha256:692eff58ac23cafc7cb099793feb00406146d187cd3ba0226809317952a9cf37 as go-alpine
 ENV GOOS=linux GOARCH=amd64 CGO_ENABLED=1
-RUN apk --update add --no-cache gcc g++ libtool git
+RUN apk --update add --no-cache gcc g++ libtool git libgcc
+WORKDIR /project
+ENTRYPOINT ["go"]
+
+FROM golang:1.11.5-stretch as go
```

Comment:
pin the hash and let's stick with 1.11.2 for stretch as well ... we bump all images to lates go later.